frontend/src/app/Settings/Facility/FacilityForm.tsx

```diff
@@ -116,7 +116,14 @@ const FacilityForm: React.FC<Props> = (props) => {
       <div className="prime-container usa-card__container">
         <div className="usa-card__header">
           <div>
-            <FontAwesomeIcon icon={"arrow-left"} color="#888" />
+            <svg
+              className="usa-icon text-base margin-left-neg-2px"
+              aria-hidden="true"
+              focusable="false"
+              role="img"
+            >
+              <use xlinkHref={iconSprite + "#arrow_back"}></use>
+            </svg>
```

Comment:
Would be cool to have this as a component eventually: 
`<Icon className="custom-class" icon={"arrow_back"} />`